agent/dns_test.go

```diff
@@ -6044,7 +6044,7 @@ func TestDNS_AddressLookup(t *testing.T) {
 	}
 	for question, answer := range cases {
 		m := new(dns.Msg)
-		m.SetQuestion(question, dns.TypeSRV)
+		m.SetQuestion(question, dns.TypeA)
```

Comment:
Leaving this comment for anyone else who may be reading this PR. We believe this test was previously incorrect because it was asking for a SRV record but getting back an A record. This change is to fix that bug and only return the type of record that is requested.